### PR TITLE
Table utility classes

### PR DESCRIPTION
**Pull request by pmario**
This PR fixes:  **[BUG] Issue in UI Text for New Journal Tiddlers and Default Tiddlers** #7037 

- this PR adds back the `tc-edit-texteditor` class
- it removes the `tag="textarea"`  parameter from the [<$edit widget](https://tiddlywiki.com/#EditWidget), since it does not support it and it causes confusion
- remove StoryList from PR

------

The rest stays the same as the original PR see: https://github.com/Jermolene/TiddlyWiki5/pull/6912

I did test it with 

- FF latest on windows
- Edge latest on windows
- FF on Android
- chrome on Android. 

Works for me.

### CODE REVIEW

File: editions/tw5.com/tiddlers/$__StoryList.tid [hunk 1]
```diff
@@ -1,5 +0,0 @@
-created: 20220728145919904
-list: Brackets Base64 HelloThere GettingStarted Community
-modified: 20220728145919904
-title: $:/StoryList
-type: text/vnd.tiddlywiki
```
Comment (saqimtiaz) [inline editions/tw5.com/tiddlers/$__StoryList.tid:5]: This should be a separate PR in order to have a cleaner and easier to follow commit history.

**Comment by saqimtiaz** [comment]
> * it removes the `tag="textarea"`  parameter from the [<$edit widget](https://tiddlywiki.com/#EditWidget), since it does not support it and it causes confusion

Can you please explain what you mean here? The $edit widget passes on all attributes to the $edit-text widget it calls. 

Are you proposing that we not use textareas but use input fields instead in ControlPanel/Basics?

File: editions/tw5.com/tiddlers/$__StoryList.tid (hunk 1 above)
Comment (pmario) [reply editions/tw5.com/tiddlers/$__StoryList.tid:5]: So you say, I should add it back and pollute the commit history even more, just to remove it with a separate PR? IMO that doesn't make sense to me

Comment (saqimtiaz) [reply editions/tw5.com/tiddlers/$__StoryList.tid:5]: Your original PR has been reverted and you are making a change unrelated to this PR. So yes, it does make sense. The story list tiddler needs to be amended to undo what is probably an erroneous commit by Jeremy but does not need to be deleted.
